Automattic/wp-calypso
# Jetpack site-less Checkout: remove Onboarding Call unused code

#### Changes proposed in this Pull Request

* A couple of weeks ago, we disabled the onboarding calls but didn't remove all the code that makes them work. This PR does precisely that. It removes everything we coded to make them work.

#### Testing instructions

* Download this PR and start Calypso Blue with `yarn start`.
* Visit `cloud.jetpack.com/pricing` and select any paid product.
* Once you're on the Checkout page, replace the `wordpress.com` part of the URL with `calypso.localhost:3000`.
* Open the JS Console, and verify you see no errors during your test.
* Complete the purchase (you can use credits).
* Once you're on the Thank You page, make sure you see no mention of the onboarding/support call.
* Enter the URL of a site that doesn't exist (we don't want the transfer to succeed).
* Once you're on the Thank You Completed page, make sure you see no mention of the onboarding/support call.
* Wait for the `Add Jetpack Backup (Daily) to your site` (the product's name might change according to the product you purchased), and make sure you see no mention of the onboarding/support call.

Related to 1200738571375997-as-1200981993809244

#### Demo
##### Thank You page
![image](https://user-images.githubusercontent.com/3418513/134575564-8713b3eb-585a-4de2-ab03-14da90f4963d.png)

##### Thank You Completed page
![image](https://user-images.githubusercontent.com/3418513/134575524-b0d12c54-3a25-4aed-a7af-31b13b7bafe4.png)

##### Email template
![image](https://user-images.githubusercontent.com/3418513/134575424-d63e1dfd-4ef5-40ad-bf56-cd8fa9951d71.png)
